maintainers/scripts/nix-guix-correlate.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
```

Comment:
alright, thanks. This is still WIP of course, more comments welcome. I will try to work a bit on using the homepage attribute to correlate packages as well.

If people are interested I could also check against for example Debian (fairly straightforward), or something like libraries.io. It might be worth exploring this topic a bit more.